src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1985,6 +2152,149 @@ await using (var context = InMemoryContentDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task GetPublicationTree_DataTables_ReleaseHasData()
+        {
+            var publicationA = new Publication
+            {
+                Title = "Publication A",
+                Slug = "publication-a",
+            };
+
+            var theme = new Theme
+            {
+                Title = "Theme A",
+                Summary = "Theme A summary",
+                Topics = new List<Topic>
+                {
+                    new()
+                    {
+                        Title = "Topic A",
+                        Publications = ListOf(publicationA)
+                    },
+                },
+            };
+
+            var release = new Release
+            {
+                Publication = publicationA,
+                ReleaseName = "2020",
+                TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                Type = ReleaseType.OfficialStatistics,
+                Published = new DateTime(2020, 1, 1),
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                await context.AddRangeAsync(
+                    theme,
+                    release,
+                    new ReleaseFile
+                    {
+                        Release = release,
+                        File = new File
+                        {
+                            Type = FileType.Data,
+                        }
+                    });
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                var service = BuildThemeService(context);
+
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataTables);
+                var publicationTree = result.AssertRight();
+
+                Assert.Single(publicationTree);
+                Assert.Equal(theme.Id, publicationTree[0].Id);
+
+                Assert.Single(publicationTree[0].Topics);
+                var topic = publicationTree[0].Topics[0];
+                Assert.Equal(theme.Topics[0].Id, topic.Id);
+
+                Assert.Single(topic.Publications);
+                var publication = topic.Publications[0];
+                Assert.Equal(publicationA.Id, publication.Id);
+                Assert.False(publication.IsSuperseded);
+                Assert.True(publication.HasLiveRelease);
+                Assert.True(publication.LatestReleaseHasData);
+                Assert.True(publication.AnyLiveReleaseHasData);
+            }
+        }
+
+        [Fact]
+        public async Task GetPublicationTree_DataTables_PreviousReleaseHasData()
```

Comment:
Is this already covered by `GetPublicationTree_DataTables_DoesFilterPublicationWithoutDataOnLatestRelease`? We might want to adjust the name to make it more readable/obvious.

These new assertions should probably be in there instead.

Reply:
I think this is already covered - I've removed this test.